### PR TITLE
Add full transcriptome

### PR DESCRIPTION
**Pull request by jashapiro**
This PR adds the reference files and index for a full transcriptome, which is the union of the `cdna` and `ncrna` transcript files from Ensembl. This is created by concatenating the two relevant fasta files after download, which is then synced to S3

### CODE REVIEW

File: workflows/rnaseq-ref-index/README.md [hunk 1]
```diff
@@ -6,13 +6,19 @@ The scripts in this directory implement a workflow for downloading and indexing
...
 ## Indexing
 
 Currently, only salmon indexing is implemented, without decoys.
-The workflow expects to find the reference cdna file at `s3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100/fasta/Homo_sapiens.GRCh38.ncrna.fa.gz` and will place the salmon index directories at `s3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100/salmon_index` with subdirectories for each type of index.
+The workflow expects to find the following reference sequence files:
+
+- **cdna** at `s3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100/fasta/Homo_sapiens.GRCh38.cdna.all.fa.gz`
+- **txome** at `s3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100/fasta/Homo_sapiens.GRCh38.txome.fa.gz`
+
+and will place the salmon index directories at `s3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100/salmon_index` with subdirectories for each type of index.
```
Comment (jaclyn-taroni) [inline workflows/rnaseq-ref-index/README.md:21]: I am almost certainly missing something (somethings?), but it is not clear to me why you would need both of these FASTA files here if `txome` is `cdna` + `ncrna`.

Comment (jashapiro) [reply workflows/rnaseq-ref-index/README.md:21]: This workflow builds indexes for both the cdna alone and the full transcriptome. I could build the txome file on the fly, I suppose, but it seemed more straightforward to make it a separate file. Willing to reconsider though!

Comment (jaclyn-taroni) [reply workflows/rnaseq-ref-index/README.md:21]: Ah, I see. I agree with that design decision. The disconnect was that I didn't understand that you are building two indices from the documentation. You do reference multiple types of index, but I was also looking at this in the context of the PR title, etc. - this is what I expected I was missing.

Comment (jashapiro) [reply workflows/rnaseq-ref-index/README.md:21]: Yup, line 40-41 defines the reference files that will be used to build indexes, so both are built, though nextflow can (and did!) skip rebuilding references that had been previously built when run with the `-resume` flag.

**Pull request merged**